tools/datafix/reput_bugs.py

```diff
@@ -14,40 +14,12 @@
 MAX_BATCH_SIZE = 500
 
 
-def main() -> None:
-  parser = argparse.ArgumentParser(
-      description="Reput all bugs from a given source.")
-  parser.add_argument(
-      "--dry-run",
-      action=argparse.BooleanOptionalAction,
-      dest="dryrun",
-      default=True,
-      help="Abort before making changes")
-  parser.add_argument(
-      "--verbose",
-      action=argparse.BooleanOptionalAction,
-      dest="verbose",
-      default=False,
-      help="Display records being operated on")
-  parser.add_argument(
-      "--source",
-      action="store",
-      dest="source",
-      default="curl",
-      help="The prefix of source_id records to reput")
-  parser.add_argument(
-      "--project",
-      action="store",
-      dest="project",
-      default="oss-vdb-test",
-      help="GCP project to operate on")
-  args = parser.parse_args()
-
-  client = datastore.Client(project=args.project)
+def main(arg) -> None:
+  client = datastore.Client(project=arg.project)
```

Comment:
Huh I just noticed this is using both datastore.Client and ndb. Can we just use ndb for the query? 

e.g. Bug.query(...)

Reply:
FYI, I think there's a degree of cargo-culting from other scripts I've written, and from memory, utilising models.py and the NDB interface versus the Datastore interface greatly complicated things from a Python dependency/packaging perspective? 

Like for starters, I think the script can't live in this directory and would need to be run from within a Pipenv as it has a dependency on the osv Python package as a result.

The initial script I'd written was to solve a particular problem, and it Just Worked, and subsequently became the skeleton for other such Datastore manipulation scripts. I didn't wish to spend the time revisiting the overall implementation.

That's the back story to the existing implementation. Happy to be corrected on the complexity.

Reply:
Hmm I kind of see pros and cons to using ndb, it's a good point that having the osv as a dependency complicates running the script, but since the script lives in this repository anyway, I don't think it matters too much since you'll have the library right there.

The main benefit of using ndb is that we get (partial) type checking, so that if we change/update the type of Bug in the future the script will fail instead of doing something unexpected, and refactoring tools can automatically update the script as well.

Reply:
> The main benefit of using ndb is that we get (partial) type checking, so that if we change/update the type of Bug in the future the script will fail instead of doing something unexpected, and refactoring tools can automatically update the script as well.
> 
> I think that's enough of a benefit to outweigh the need to have the python environment loaded when running the script.

It certainly sounds like a compelling argument. Am I right in the code would need to move under the osv directory, or am I misunderstanding that nuance of Python dependencies?

I'd initially wanted to continue using the tools directory for standalone, you know, tools... These have no business winding up in the osv Python package.